tests/models/test_serialization.py

```diff
@@ -0,0 +1,51 @@
+import typing as tp
+from tempfile import NamedTemporaryFile
+
+import pytest
+from implicit.als import AlternatingLeastSquares
+from implicit.nearest_neighbours import ItemItemRecommender
+
+try:
+    from lightfm import LightFM
+except ImportError:
+    LightFM = object  # it's ok in case we're skipping the tests
+
+import sys
+
+from rectools.models import (
+    ImplicitALSWrapperModel,
+    ImplicitItemKNNWrapperModel,
+    LightFMWrapperModel,
+    PopularInCategoryModel,
+    load_model,
+)
+from rectools.models.base import ModelBase
+
+from .utils import get_final_successors
+
+MODEL_CLASSES = [
+    cls
+    for cls in get_final_successors(ModelBase)
+    if cls.__module__.startswith("rectools.models") and not (sys.version_info >= (3, 12) and cls is LightFMWrapperModel)
```

Comment:
What's wring with LightFM and 3.12?

Reply:
they are incompatible with each other (we also skip all lightfm tests for 3.12)